### DISCUSSION TITLE
Introduce "overflow: overlay"

### DISCUSSION THREAD

**Discussion by dnlsndr** [tailwindlabs/tailwindcss#10569] [Ideas]
It seems that the `overflow: overlay` styling hasn't been added to the tailwind library yet.
It's currently not possible to use a inline-value such as `overflow-[overlay]` AFAIK.
The CanIUse coverage of that feature is at 76% but it seems that Safari converts it to `overlay: auto`, and Firefox does not yet implement it.

So maybe it might not be enough coverage for the base package, but would it make sense to create a `next` plugin where these "new" features are implemented, even it they might not work in all browsers?

**Comment by adamwathan** [comment]
Hey! For anything that’s not included yet we recommend the arbitrary properties feature:

```html
<div class=“[overflow:overlay]”>
```

I’m sure we’ll add it eventually as it becomes a more supported and popular CSS feature 👍🏻

**Comment by agm1984** [comment]
Just looking for this today to prevent scrollbar from pushing the content to the left of the scrollbar creating extra padding. 

Looks like they removed it though: https://caniuse.com/css-overflow-overlay
